.drone.yml

```diff
@@ -4,17 +4,6 @@ name: default
 
 steps:
 
-- name: chart-lint
-  image: quay.io/helmpack/chart-testing:v2.3.3
-  commands:
-  - ct lint --config test/ct.yaml
-  when:
-    branch:
-      - master
-    event:
-      - push
-      - pull_request
-
```

Comment:
@rimusz is the `ct lint --config test/ct.yaml` command replaced somewhere in the e2e-kind script? I don't see it exactly but it looks close, so just making sure

Reply:
this does run the chart lint part which doesn't need kind.
`e2e-kind.sh` has `ct install --config test/ct.yaml`
Also we can do just in `kind` `ct lint-and-install --config test/ct.yaml` which will do both.
Let me know if you this change.

Reply:
https://github.com/gomods/athens/pull/1225/commits/12901a8d0550c1a3bb43ed09a589429d81757769#diff-37eda1430799e62f92596aa54fae0596R85